tests/unit/s2n_client_psk_extension_test.c

```diff
@@ -316,32 +425,102 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_psk_parameters_free(&params));
     }
 
-    /* Test: s2n_client_psk_recv_identity_list */
-    {
-        const uint8_t single_wire_identity[] = {
-                TEST_BYTES_SIZE,        /* identity size */
-                TEST_BYTES,             /* identity */
-                0x00, 0x00, 0x00, 0x00, /* ticket_age */
-        };
+    /* Test: s2n_choose_psk */
+    { 
```

Comment:
Shouldn't these tests be in s2n_psk_test, since that's where the callback lives?